dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
The fabfile uses `apt-get install` which is the package manager for debian/ubuntu, but will never work for any redhat variant, arch, gentoo, etc.

Using docker you can run a `ubuntu` container on any platform that supports docker (any linux, osx, or windows).